package.json

```diff
@@ -38,7 +38,7 @@
     "ci:bump-form-engine": "yarn up @ohri/openmrs-ohri-form-engine-lib@next"
   },
   "devDependencies": {
-    "@openmrs/esm-framework": "4.1.1-pre.544",
+    "@openmrs/esm-framework": "next",
```

Comment:
There a number of speed fixes happening at the community level so we should maintain this for now.